Extra/Shaders/NewVegasReloaded/Shaders/Includes/Tonemapping.hlsl

```diff
@@ -215,47 +215,91 @@ float ColToneC(float hdrMax, float contrast, float shoulder, float midIn, float
 }
 
 // General tonemapping operator, p := {contrast,shoulder,b,c}.
-float ColTone(float x, float4 p)
+float ColTone(float3 x, float4 p)
 {
-    float z = pows(x, p.r);
+    float3 z = pows(x, p.r);
     return z / (pows(z, p.g) * p.b + p.a);
 }
 #define CMAX 1.6e+6f
+#define EPS 1e-6f
 float3 VTLottes(float3 color, float contrast, float midOut, float midIn, float hdrMax, float shoulder, float crossTalk)
```

Comment:
are we no longer using VTLottes? It could be good here to have a link to vtastek's gist for sourcing

Reply:
Renamed AMDLottes back to VTLottes and removed the larger non-optimised code I commented out, since AMDLottes is just optimised VTLottes. Should restore the credits and trim the code